src/clang.rs

```diff
@@ -387,16 +387,26 @@ impl Cursor {
 
     /// Given that this cursor's referent is a function, return cursors to its
     /// parameters.
-    pub fn args(&self) -> Vec<Cursor> {
+    pub fn args(&self) -> Option<Vec<Cursor>> {
+        // XXX: We might want to use and keep num_args
```

Comment:
Could you elaborate on this, please? Does this mean we should be returning an iterator instead of a Vec? Or that there's some sort of iterator that we can "build on" to write this method?